content/docs/dvclive/api-reference/live/log.md

```diff
@@ -13,45 +13,55 @@ from dvclive import Live
 
 live = Live()
 
-live.log("loss", 0.9)
+live.log("loss", 0.1)
+live.log("acc", 0.9)
 ```
 
 ## Description
 
-If `summary` is True, `live.log(name, val)` will update the
-[summary](/doc/dvclive/get-started#summary) with the latest value logged.
+If [`summary`](/doc/dvclive/api-reference/live/#parameters) is enabled, on each
+`live.log(name, val)` call DVCLive will add a `name` entry in `{path.json}` with
+the corresponding `val`:
 
-💡 The summary `{path}.json` is usable by `dvc metrics`.
```

Comment:
As mentioned in the getting started page, it's hard to visualize the directory structure of the outputs. For each log method, it would be helpful to show the directory structure (maybe separately with and without steps where applicable). For example:

```
$ tree
dvclive.json
dvclive/scalars/acc.tsv
```